sdk/containerregistry/Azure.ResourceManager.ContainerRegistry/src/autorest.md

```diff
@@ -9,7 +9,7 @@ csharp: true
 library-name: ContainerRegistry
 namespace: Azure.ResourceManager.ContainerRegistry
 # default tag is a preview version
-require: https://github.com/Azure/azure-rest-api-specs/blob/d37b3db5ec88a9fc2afcae36bf8d2004b1c5b537/specification/containerregistry/resource-manager/readme.md
+require: https://github.com/Azure/azure-rest-api-specs/blob/8a28143c7271d4496296ed47f70c3cb5a9981e57/specification/containerregistry/resource-manager/readme.md
```

Comment:
Is this change manually or generated? Seems no content change between them.

Reply:
I fixed the auth value in the swagger repo examples, so here need to update the link